.github/workflows/production-workflow.yml

```diff
@@ -0,0 +1,53 @@
+name: "Production Build"
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
```

Comment:
Remove pull_request triggers
It only should be triggered when there is a push to main
If not, it will deploy the app on every PR action
My mistake